packages/odyssey-react/src/components/Toast/Toast.test.tsx

```diff
@@ -67,7 +67,11 @@ describe("Toast", () => {
   it("restricts children prop via types and does not render them", () => {
     render(
       // @ts-expect-error never type for children
```

Comment:
The change to `heading` pushes this over the line length limit, so Prettier breaks it into a block, rather than a single line.

`@ts-expect-error` and `@ts-ignore` don't currently support blocks, so this now results in an error.


@jeffbelser-okta best way to proceed here?

Reply:
```
      <Toast
        heading={heading}
        dismissButtonLabel={dismissLabel}
        // @ts-expect-error never type for children
        children="child"
      />
```

Reply:
Thanks - this wasn't working locally, but CI is happy. I'll investigate my linting setup and make sure my configs haven't diverged somehow.